.github/actions/setup-spack-stack/action.yaml

```diff
@@ -208,11 +208,11 @@ runs:
       spack external find mpich
       spack external find openmpi
       spack external find perl
-      spack external find python@3
+      spack external find python
```

Comment:
Why is it okay to remove the python version 3 qualifier? Isn't it safer with that qualifier?

Reply:
Spack no longer allows this syntax, it complains about an unknown package `python@3` and same for `qt@5`. It should be safe still, because it will always prefer the latest Python (=3.x.y), even if it finds additional/older versions.

For qt, even if it finds qt@5 and qt@6 (which it won't because we don't install it in the CI builds), ecflow specifically requires qt@5, so that is ok as well.

Reply:
I see. Thanks for the explanation!